src/bindings/slash_commands.ts

```diff
@@ -27,7 +27,7 @@ function commandConnect(): AppBinding {
         description: 'Connect your Zendesk account',
         icon: ZDIcon,
         call: {
-            url: Routes.App.BindingPathConnect,
+            path: Routes.App.BindingPathConnect,
```

Comment:
Looking at the `call` struct here, https://github.com/mattermost/mattermost-plugin-apps/blob/master/apps/call.go#L35-L42 we are still using `url` as the field.  Are we going to change this in master?

Reply:
https://github.com/mattermost/mattermost-plugin-apps/pull/77 will update that.